routers/init.js

```diff
@@ -8,18 +8,26 @@
     It also needs to be included inside of app.js, next to the other 'app.use' statements.
  */
 
-const web = require('./web')
-const login = require('./login')
-const signup = require('./signup')
-const dashboard = require('./dashboard')
-const profile = require('./profile')
-const error = require('./error')
+// const web = require('./old_routers/web')
```

Comment:
Same comment as above, perhaps we should consider removing these instead of just commenting them out.